### PR TITLE
createWriteStream

### PR DESCRIPTION
**Pull request by staltz**
1st test, 2nd fix

### CODE REVIEW

File: test/createHistoryStream.js [hunk 1]
```diff
@@ -69,6 +70,42 @@ test('No values', (t) => {
   )
 })
 
+test('createWriteStream', (t) => {
+  const rando = ssbKeys.generate()
```
Comment (arj03) [inline test/createHistoryStream.js:74]: :-)

Comment (staltz) [reply test/createHistoryStream.js:74]: This should fix the "rando problem"

**Review by arj03 — APPROVED** [review]

**Comment by arj03** [comment]
Sweet. Nice and simple.

**Comment by staltz** [comment]
I might later make a PR to use `pull-drain-gently` inside createWriteStream (obviously this would be guarded by a check whether `config.maxCpu` exists), but if and only if I discover it helps manyverse during initial replication.

**Pull request merged**